api/features/views.py

```diff
@@ -394,10 +374,6 @@ def create(self, request, *args, **kwargs):
                 status=status.HTTP_201_CREATED,
                 headers=headers,
             )
-        else:
-            logger.error(serializer.errors)
```

Comment:
We are changing the response structure for 400 here though, thoughts @matthewelwell